SCAutolib/models/file.py

```diff
@@ -123,6 +134,50 @@ def set(self, key: str, value: Union[int, str, bool], section: str = None):
             logger.debug(f"Old value in section [{section}] {key}={previous}")
             logger.debug(f"New value in section [{section}] {key}={value}")
 
+    def get(self, key, section: str = None, separator: str = "="):
+        """
+        Method would return the value of the key in section (if set). If the
+        file do not support sections (section for this method is not provided),
+        then file would be parsed in a simple way with `.readlines()` method and
+        using separator each line would be split into two parts: key and a
+        value. If key is matched, the value is striped and returned.
```

Comment:
We should explicitly mention that first match is returned or rewrite this method (see below).